app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -91,9 +91,19 @@ class EditSectionActivity : BaseActivity(), ThemeChooserDialog.Callback, EditPre
 
     private var actionMode: ActionMode? = null
 
+    private val movementMethodWithLogin = LinkMovementMethodExt { url: String ->
+        if (url == "https://#login") {
+            val loginIntent = LoginActivity.newIntent(this@EditSectionActivity, LoginActivity.SOURCE_EDIT)
+            requestLogin.launch(loginIntent)
```

Comment:
Not sure if we still need this since the license text does not contain any `#login` anymore.

Reply:
done!